src/features/collapsible-sidebar/CollapsibleSidebar.scss

```diff
@@ -111,6 +111,24 @@ $left-sidebar-z-index: 40 !default;
     }
 }
 
+@include breakpoint($small-screen) {
+    .Body.is-responsive-web {
+        .bdl-CollapsibleSidebar-wrapper {
+            position: initial;
```

Comment:
I understand from this [thread](https://github.com/box/box-ui-elements/pull/2882#discussion_r865391641) there's choppiness if we set `width: 100%;` for `.bdl-CollapsibleSidebar-nav ul`.

But in combination with the new calc solution for `.bdl-CollapsibleSidebar-nav ul`, are we able to set `width: 100%` for the wrapper now?

Since `CollapsibleSidebar` is used in other apps such as Admin Console and Dev Console, it'd be nice if this change could make it responsive there as well. Thus, I don't think we can rely on the `position` of the parent element (in this case `in-app-messenger-provider`).

Reply:
Okay, thanks for mentioning this. Take a look at the latest commits. I ended up with a solution where the `wrapper` was able to keep its position=relative. All I had to do was set `.bdl-CollapsibleSidebar.is-expanded` to width=100vw.